bentoml/detectron.py

```diff
@@ -208,33 +218,42 @@ def num_replica(self) -> int:
             return len(self.resource_quota.gpus)
         return 1
 
-    # pylint: disable=arguments-differ,attribute-defined-outside-init
-    def _setup(self) -> None:  # type: ignore[override]
+    # pylint: disable=attribute-defined-outside-init
+    def _setup(self) -> None:
         if self.resource_quota.on_gpu:
             device = "cuda"
         else:
             device = "cpu"
         self._model = load(self._tag, device, self._model_store)
         self._predict_fn = getattr(self._model, self._predict_fn_name)
 
-    # pylint: disable=arguments-differ
-    def _run_batch(  # type: ignore[override]
-        self, input_data: t.Union[np.ndarray, torch.Tensor]
-    ) -> np.ndarray:
-        images = np.split(input_data, input_data.shape[0], 0)
-        images = [image.squeeze(axis=0) for image in images]
-        if isinstance(input_data, np.ndarray):
-            images = [torch.from_numpy(image) for image in images]
+    def _run_batch(
+        self,
+        *args: t.Union["np.ndarray[t.Any, np.dtype[t.Any]]", torch.Tensor],
+        **kwargs: t.Any,
```

Comment:
does detectron model's `__call__` accept kwargs? if not, we don't need to add this

Reply:
we actually want to support kwargs in this case. https://github.com/facebookresearch/detectron2/blob/main/projects/Panoptic-DeepLab/panoptic_deeplab/panoptic_seg.py

Reply:
```suggestion
        **kwargs: t.Union["np.ndarray[t.Any, np.dtype[t.Any]]", torch.Tensor],
```